language/tools/move-cli/src/test.rs

```diff
@@ -135,3 +136,28 @@ pub fn run_one(args_path: &Path, cli_binary: &str) -> anyhow::Result<()> {
         Ok(())
     }
 }
+
+pub fn run_all(args_path: &Path, cli_binary: &str) -> anyhow::Result<()> {
+    let mut test_total = 0;
+    let mut test_passed = 0;
+    for entry in WalkDir::new(args_path)
+        .into_iter()
+        .filter_map(Result::ok)
+        .filter(|e| e.file_type().is_file())
+    {
+        let fname = entry.file_name().to_string_lossy();
+        if fname == "args.txt" {
```

Comment:
Nit: add this check to the `filter`?

Reply:
I thought of that before, but since later we need `fname` in the `eprintln`, I decided to leave this check out of the filter.

Reply:
Ah, that makes sense. Although: we might want to print the whole path (or at least the parent dir) at 155 given that `fname` will always be "args.txt" (with no prefix).

Reply:
Ah, silly of me. You are right. We should print `entry.path()`. Then this one should go to the filter.